Source/FullScan/Mixins/Frame.lua

```diff
@@ -60,18 +62,56 @@ function AuctionatorFullScanFrameMixin:UnregisterForEvents()
   FrameUtil.UnregisterFrameForEvents(self, FULL_SCAN_EVENTS)
 end
 
+function AuctionatorFullScanFrameMixin:CacheScanData()
+  self:ResetData()
+
+  local left = C_AuctionHouse.GetNumReplicateItems()
+
+  for i = 0, C_AuctionHouse.GetNumReplicateItems() - 1 do
+    local info = { C_AuctionHouse.GetReplicateItemInfo(i) }
+    local link = C_AuctionHouse.GetReplicateItemLink(i)
+
+    if not info[18] then
```

Comment:
Not strictly necessary, but I like to create constants for these index values so its easier to understand what the code is doing (and also easier to update if the order of return values changes, which Blizzard has done before).